script/src/verify.rs

```diff
@@ -552,7 +553,7 @@ mod tests {
             runner: Runner::default(),
         };
         let verifier = TransactionScriptsVerifier::new(&rtx, &data_loader, &config);
-
+        verifier.verify(100).unwrap();
```

Comment:
remove this line?

Reply:
ops, added for debug, forgot to delete...